lib/charms/istio_beacon_k8s/v0/service_mesh.py

```diff
@@ -83,8 +76,11 @@ def __init__(self, *args):
 ```
 
 This example creates two policies:
-- When related over the `metrics` relation allow the related application to `GET` this application's `/metrics` endpoint on the specified port
-- When related over the `data` relation allow the relation application to `GET` this application's `/data` endpoint on the specified port
+- An unit policy - When related over the `metrics` relation allow the related application to access the units (via the unit/workload ip or FQDN) of this application without any restriction.
```

Comment:
I think mentioning IP or FQDN might be confusing here.  What about just saying the app's service or workload?

```suggestion
- A unit policy - When related over the `metrics` relation, allow the related application to access this application's unit Pods directly without any restriction.
```